dbms/src/WindowFunctions/tests/gtest_first_value.cpp

```diff
@@ -0,0 +1,141 @@
+// Copyright 2023 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Interpreters/Context.h>
+#include <TestUtils/ExecutorTestUtils.h>
+
+namespace DB::tests
+{
+// TODO Tests with frame should be added
+class FirstValue : public DB::tests::ExecutorTest
```

Comment:
And add test case in `tests/fullstack-test/mpp/window.test`?

Reply:
> And add test case in `tests/fullstack-test/mpp/window.test`?

okk, I will add it.

Reply:
> And add test case in `tests/fullstack-test/mpp/window.test`?

I will add window ft for `first_value` and `last_value` in a new pr after open the pushdown switch in tidb.

Reply:
Why not just add it to this pr, and it can be executed successful before tidb supports it.

Reply:
> Why not just add it to this pr, and it can be executed successful before tidb supports it.

okk, I will add it.